packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -761,12 +761,15 @@ export class AppStudioPluginImpl {
   > {
     let tabEndpoint, tabDomain;
     if (isArmSupportEnabled()) {
-      if (localDebug) {
+      // getConfigForCreatingManifest is called in post-provision and validate manifest
+      // only in post stage, we find the value from arm output.
+      // Here is a walk-around way, try to get from arm output first and then get from ctx config.
```

Comment:
Why do we `try to get from arm output first and then get from ctx config`? Should the value always can be got from arm output?

Reply:
Because this method will be called both in provision stage and no-provision stage.
arm output only exists in provision stage.
Here is a walk-around way to handle it. 
Ideally we should withdraw the code about provision stage.
I think we can refactor it when our design about arm output and ctx config is done.